notebooks/Data_visualization/Data_visualizations.ipynb

```diff
@@ -0,0 +1,474 @@
+{
```

Comment:
<p>We use Seaborn as the go-to package within our team for cleaner visualizations. Can we also use Seaborn here to support the examples?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/data-science-workflows/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>+1 </p>

Reply:
<p>+1, <code>seaborn</code>  is the library we standardized to use, so would be great to mention that</p>